Configure

```diff
@@ -391,6 +391,7 @@ my @disablables = (
     "bf",
     "blake2",
     "buildtest-c++",
+    "bulk",
```

Comment:
This option needs documenting in INSTALL.md.

Reply:
Good point - I've now documented is a follows:
```
### no-bulk

Build only a minimal set of features.
```
Note that I resisted enumerating everything it implies (also because that long list would need to be kept in sync with  `Configure`).

Reply:
You should say "Used for the project's CI builds" or something like that.

Reply:
Yes, I've added:
```
This is used for CI build tests of the project.
```